source/_components/ifttt.markdown

```diff
@@ -34,9 +34,10 @@ You can then consume that information with the following automation:
 ```yaml
 automation:
   trigger:
-    event: ifttt_webhook_received
+    platform: event
```

Comment:
Did you test this change?

Reply:
Yes it wouldn't work the way it was. I tested with sleep as android using.

```
- id: '1525567357516'
  alias: IFTTT
  trigger:
    platform: event
    event_type: ifttt_webhook_received
    event_data:
     action: call_service
  action:
    service_template: "{{ trigger.event.data.service }}"
    data_template:
      entity_id: "{{ trigger.event.data.entity_id }}"
```

Reply:
The existing version doesn't have the `platform: event`, it has `event: ifttt_webhook_received` and does not work as it. This change is needed. See the [event trigger documentation](https://www.home-assistant.io/docs/automation/trigger/#event-trigger).